README.md

```diff
@@ -671,32 +743,38 @@ only marking here that you are likely to hear both.
     Say you take three months of parental leave in your first year.
     Does your cliff extend to 15 months?
 
-- 🄳 Vesting *can* be triggered outside of the vesting schedule.
-  [**Acceleration**](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/) dictates that
-  vesting will be triggered if a company is sold (**single trigger**) or if it’s sold and
-  you’re fired (**double trigger**).
+- 🄳 Vesting may be triggered outside of the vesting schedule, depending on the terms of an
+  agreement. [Accelerated vesting](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/)
+  is a stipulation that vesting will be triggered if a company is sold (**single trigger**)
+  or if it’s sold and you’re fired (**double trigger**).
   - ☝️ Acceleration is common for founders and not so common for employees.
     It’s worth understanding acceleration and triggers in case they show up in your option
     agreement, but these may not be something you can negotiate unless you are going to be in
     a key role.
+- Companies may impose additional restrictions on stock that is vested.
+  For example, your shares are very likely subject to a right of first refusal.
+  And it can happen that companies reserve the right to repurchase vested shares in certain
+  events.
+  - 🚧 Can we give any examples here?
 
 ### How options expire
 
 🄳 The **exercise window** is the period during which you can buy shares at the strike
-price. Options are only exercisable for a fixed period of time, typically seven to ten
-years as long as you’re working for the company.
+price. Options are only exercisable for a fixed period of time, until they **expire**,
+typically seven to ten years as long as you’re working for the company.
 But this window is not always open.
 
-- ❗ Options can expire after you quit working for the company.
-  Often, the expiration is **90 days** after termination of service, making the options
-  effectively worthless if you cannot exercise before that point.
+- ❗ **Expiration after termination**: Options can expire after you quit working for the
+  company. Often, the expiration is **90 days** after termination of service, making the
+  options effectively worthless if you cannot exercise before that point.
   We’ll talk more about this, but remember that you’ll need to understand costs and tax
-  liabilities and try to plan ahead!
+  liabilities of exercise and to plan ahead!
   In fact, you should find out about this when you start, or better yet, before you sign an
   offer letter.
-- 🔹 Recently (since around 2015) a few companies are finding ways to keep the exercise
-  window open for years after leaving a company, promoting this practice as fairer to
-  employees. Companies with
+- 🔹 **Longer exercise windows**: Recently (since around 2015) a few companies are finding
```

Comment:
These are also offered by some companies in the event of death (when your options are transferred to your estate) or when leaving for injury or disability. Will add.